content/3_Silver/Flood_Fill.mdx

```diff
@@ -326,7 +350,7 @@ public static void main(String[] args){
 
```

Comment:
I think it may be clearer if we have the red square backtrack instead of jumping from the bottom square to the top. It may not be intuitive why the red square knows to go there.

Reply:
The only problem would be that this would add like 4 more squares, making it really long. Maybe we can have 2 or 3 square on each row?

Reply:
this can be done in separate PR